### DISCUSSION TITLE
[Feature] Add statistics on downloads and views

### DISCUSSION THREAD

**Discussion by rekcodocker** [immich-app/immich#9040] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have a feature request to keep count of how many times albums and images were viewed and downloaded.

I use Immich as a photographer, to distribute photos from shoots.
Statistics would help to see if the client has downloaded the images, and also to see which images are the most popular.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by Sisis700** [comment]
El mar, 23 abr 2024 16:29, rekcodocker ***@***.***> escribió:

> I have searched the existing feature requests to make sure this is not a
> duplicate request.
>
>    - Yes
>
> The feature
>
> I have a feature request to keep count of how many times albums and images
> were viewed and downloaded.
>
> I use Immich as a photographer, to distribute photos from shoots.
> Statistics would help to see if the client has downloaded the images, and
> also to see which images are the most popular.
> Platform
>
>    - Server
>    - Web
>    - Mobile
>
> —
> Reply to this email directly, view it on G
> <https://github.com/immich-app/immich/discussions/9040>ithub
>

, or unsubscribe

**Comment by johnmmcgee** [comment]
This would be a handy feature.

**Comment by lukefind** [comment]
I could really do with this feature.. or a guide for this in Grafana

**Comment by nOrphf** [comment]
Yes please. Could be awesome.

**Comment by mzpx70** [comment]
It would be a great source of usage info. As an alternative solution some API provide statistical data for external loggers/apps (eg. Grafana for visualization)
Thanks!
